crates/medmodels-core/src/medrecord/querying/selection.rs

```diff
@@ -939,7 +939,11 @@ mod test {
 
     #[test]
     fn test_select_edges_edge() {
-        let medrecord = create_medrecord();
+        let mut medrecord = create_medrecord();
+
+        medrecord
+            .add_group("test".into(), None, Some(vec![0]))
+            .unwrap();
```

Comment:
Invalid test for this needed?

Reply:
adding the group here is not part of the actual test, only the setup.